src/test/java/org/eclipse/emfcloud/jackson/tests/annotations/JsonRawValueTest.java

```diff
@@ -0,0 +1,58 @@
+package org.eclipse.emfcloud.jackson.tests.annotations;
```

Comment:
Please add a copyright header with EPL-2.0 OR MIT (similar to the other headers). You can add your name or affiliation at the top if you want.